lib/rubocop/cop/style/hash_each_methods.rb

```diff
@@ -69,6 +97,23 @@ def register_kv_offense(target, method)
           end
         end
 
+        def exist_unused_block_arg?(node, block_arg_source)
```

Comment:
Should this be
```suggestion
        def unused_block_arg_exists?(node, block_arg_source)
```
? Or alternatively,


```suggestion
        def ignores_block_arg?(node, block_arg_source)
```